src/redux/Auth/auth-api.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+export const signin = async (data) => {
+  return await axios.post('api/user/signin', data);
+}
```

Comment:
Also add cancel tokens in all requests and understand the reason of using cancel tokens